main.py

```
@@ -0,0 +1,375 @@
+async def _cms_get(path: str, params: dict, client: httpx.AsyncClient) -> tuple[dict, int]:
+    try:
+        response = await client.get(
+            path,
+            params=params,
+        )
+        response.raise_for_status()
+    except httpx.HTTPStatusError as e:
+        logger.warning("CMS HTTP error %s for %s",
+                       e.response.status_code, path)
+        raise HTTPException(
+            status_code=e.response.status_code, detail=e.response.text)
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)